data/json/vitrification/vitrified_snippets.json

```diff
@@ -0,0 +1,145 @@
+[
+  {
+    "type": "snippet",
+    "category": "Vitrifaction_Threshold",
+    "text": [
+      "You arrive at the farmhouse, after what feels like an eternity. It's not quite what you were expecting, but it's very peaceful."
+    ]
+  },
+  {
+    "type": "snippet",
+    "category": "Vitrifaction_Blocked",
+    "text": [ "There's nothing that way but a plane of featureless black." ]
+  },
+  {
+    "type": "snippet",
+    "category": "Vitrifaction_Ambient_1",
+    "text": [
+      "With all this glass, you expected it to be brighter here. It's not. That's okay, it would have hurt your eyes otherwise.",
+      "A gust of wind blows across the farm. Glass chimes and keens in the breeze.",
+      "You are suddenly very aware of your heartbeat, but only for a moment.",
+      "The glass around you isn't sparkling; it doesn't reflect. You're not sure what it's doing instead.",
+      "It's very cold here.",
+      "You didn't realize there had been an ache at the base of your neck all day until just now, when you notice it's gone.",
+      "There's nothing here, really. How long has it been since you've felt truly safe?",
+      "You find yourself wondering what the glass tastes like.",
+      "The glass is solid under your feet. It supports you completely. Why can't you feel yourself standing?"
+    ]
+  },
+  {
+    "type": "snippet",
+    "category": "Vitrifaction_Ambient_2",
+    "text": [
+      "The edges of the glass are very sharp. But that is the nature of glass. Isn't it?.",
+      "You think, for a moment, that you might start flying. You don't, but it doesn't trouble you much.",
+      "Somewhere, inside a ribcage, something is beating very fast. Something about that strikes you as very funny.",
+      "How long has it been since you blinked? It probably doesn't matter.",
+      "You spend a moment looking at your reflection in the glass. What a nice smile you have!",
+      "How very smooth it all is. Even the sharp parts; especially the sharp parts. Pristine, and perfect.",
+      "You're so glad you decided to visit.",
+      "You were shivering, a little bit ago. It's stopped now. You must have warmed up.",
+      "Everything seems so much more real than usual. It's all right there. Reach out; touch it."
```

Comment:
```suggestion
      "The edges of the glass are very sharp.  But that is the nature of glass. Isn't it?.",
      "You think, for a moment, that you might start flying.  You don't, but it doesn't trouble you much.",
      "Somewhere, inside a ribcage, something is beating very fast.  Something about that strikes you as very funny.",
      "How long has it been since you blinked?  It probably doesn't matter.",
      "You spend a moment looking at your reflection in the glass.  What a nice smile you have!",
      "How very smooth it all is.  Even the sharp parts; especially the sharp parts.  Pristine, and perfect.",
      "You're so glad you decided to visit.",
      "You were shivering, a little bit ago.  It's stopped now.  You must have warmed up.",
      "Everything seems so much more real than usual.  It's all right there.  Reach out; touch it."
```